NewForce-Cohort-4/tabloid-full-stack-coral
# Wh view tags

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes ticket # 13

## Type of change

_Please delete options that are not relevant_.


# How Has This Been Tested?

1. Login as any user
2. Navigate to "Tag Management" in navbar
3. Ensure 4 tags are being displayed in alphabetical order

# Checklist:

- [] New and existing unit tests pass locally with my changes
